libs/template/templates/default-python/template/{{.project_name}}/README.md.tmpl

```diff
@@ -0,0 +1,36 @@
+# {{.project_name}}
+
+The '{{.project_name}}' project was generated using the default-python template.
+
+## Getting Started
+
+1. Install the Databricks CLI from https://docs.databricks.com/en/dev-tools/cli/databricks-cli.html
+
+2. Authenticate to your Databricks workspace:
+    ```
+    $ databricks configure
+    ```
+
+3. To deploy a development copy of this project, type:
+    ```
+    $ databricks bundle deploy --target dev
+    ```
+    (note that "dev" is the default target, so the `--target` parameter
+    is optional here)
+
+    This deploys everything that's defined for this project.
+    For example, the default template would deploy a job called
+    `[dev yourname] {{.project_name}}-job` to your workspace.
+    You can find that job by opening your workpace and clicking on "Workflows."
+
+4. Similarly, to deploy a production copy, type:
+   ```
+   $ databricks bundle deploy --target prod
+   ```
+
+5. Optionally, install developer tools such as the VS Code plugin from
+   https://docs.databricks.com/en/dev-tools.
+
+6. For documentation on the Databricks Asset Bundles format used
+   for this project, and for CI/CD configuration, see
+   https://docs.databricks.com/en/dev-tools/bundles.
```

Comment:
```suggestion
6. For documentation on the Databricks asset bundles format used
   for this project, and for CI/CD configuration, see
   https://docs.databricks.com/dev-tools/bundles/index.html.
```

Reply:
"Asset Bundles" should never be capitalized.

Adding "index.html" to make the URL more explicit.

Reply:
Do we really need index.html? Seems like it just makes the URL longer?

In any case, I'll accept the suggestion.